.github/workflows/tests.yml

```diff
@@ -229,25 +229,16 @@ jobs:
           cd bottles
           brew test-bot ${{ needs.setup_tests.outputs.test-bot-formulae-args }}
 
-      - name: Failures summary for brew test-bot ${{ needs.setup_tests.outputs.test-bot-formulae-args }}
+      - name: Failures Summary and Bottle Result
         if: always()
-        run: |
-          touch bottles/steps_output.txt
-          cat bottles/steps_output.txt
-          rm bottles/steps_output.txt
+        uses: Homebrew/actions/failures-summary-and-bottle-result@master
 
       - name: Output brew linkage result
```

Comment:
What about this step? Shouldn't it be done inside the action too?

Reply:
Looking back at this comment - let me know what your plans are for this after we merge this.